src/main/java/org/folio/edge/sip2/BootstrapVerticle.java

```diff
@@ -0,0 +1,58 @@
+package org.folio.edge.sip2;
+
+import io.vertx.config.ConfigRetriever;
+import io.vertx.config.ConfigRetrieverOptions;
+import io.vertx.core.AbstractVerticle;
+import io.vertx.core.DeploymentOptions;
+import io.vertx.core.Promise;
+import io.vertx.core.json.JsonObject;
+import java.util.Objects;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+public class BootstrapVerticle extends AbstractVerticle {
+  private final Logger log = LogManager.getLogger();
+  private String deploymentID = null;
+  
+  @Override
+  public void start(Promise<Void> startFuture) {
+
+    log.info("Bootstrapping sip2, retrieving config from:\n {}", () -> config().encodePrettily());
```

Comment:
No secrets in config, just okapiUrl, bucket name, & region

Reply:
What if someone wants to use another provider, like Kubernetes or Vault? It looks like some of these config providers provide mechanisms where a password could be in the `stores` JSON. Maybe we can just output the store `type` or maybe we take out the `config` JSON in case there are secrets in there?

Reply:
I set the config to ******** in the logs